Framework/DataHandling/src/SaveGSS.cpp

```diff
@@ -715,7 +715,11 @@ std::map<std::string, std::string> SaveGSS::validateInputs() {
   std::map<std::string, std::string> result;
 
   API::MatrixWorkspace_const_sptr input_ws = getProperty("InputWorkspace");
-
+  if (!input_ws) {
+    result.emplace("InputWorkspace",
```

Comment:
Is there an reason to use `emplace` instead of `results["InputWorkspace"]= "The input workspace cannot be a GroupWorkspace."`?